### PR TITLE
Jordan/deps

### PR DESCRIPTION
**Pull request by jbibla**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ module.exports = {
         extensions: [`.js`, `.vue`, `.css`]
       },
       plugins: [
+        new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```
Comment (faboweb) [inline vue.config.js]: why did you add this and what does it do?

Comment (jbibla) [reply vue.config.js]: significantly reduces the size of moment.js 

https://github.com/jmblog/how-to-optimize-momentjs-with-webpack#how-to-optimize-momentjs-with-webpack

Comment (faboweb) [reply vue.config.js]: I just checked and we do not localize dates. Shouldn't we do that? Not everyone

Comment (jbibla) [reply vue.config.js]: yeah, maybe but this has not come up as an issue and we don't really have an internationalization strategy right now. when we do localization we can re-add the locales we need - but for now, I think we should just remove this unused code.

**Comment by jbibla** [comment]
we needed to upgrade cosmos-keys and cosmos-ledger, right?

File: vue.config.js (hunk 1 above)
Comment (colw) [reply vue.config.js]: This is a good moment to point out that the perfect place to 

> significantly reduces the size of moment.js
> 
> https://github.com/jmblog/how-to-optimize-momentjs-with-webpack#how-to-optimize-momentjs-with-webpack

This is a great moment to point out that information like this belongs in the commit message :)

**Comment by jbibla** [comment]
i haven't tested if the ledger actually works here yet.
